minecraft-server/README.md

```diff
@@ -529,6 +529,72 @@ Just change it with `SPONGEBRANCH`, such as:
     $ docker run -d -v /path/on/host:/data ... \
         -e TYPE=SPONGEVANILLA -e SPONGEBRANCH=EXPERIMENTAL ...
 
+## Running a Fabric Server
+
+Enable Fabric server mode by adding a `-e TYPE=FABRIC` to your command-line.
+By default the container will run the latest version of [Fabric server](http://fabricmc.net/use/)
+but you can also choose to run a specific version with `-e FABRICVERSION=0.5.0.32`.
+
+    $ docker run -d -v /path/on/host:/data -e VERSION=1.14.3 \
+        -e TYPE=FABRIC -e FABRICVERSION=0.5.0.32 \
+        -p 25565:25565 -e EULA=TRUE --name mc itzg/minecraft-server
+
+To use a pre-downloaded Forge installer, place it in the attached `/data` directory and
```

Comment:
Should this be
```suggestion
To use a pre-downloaded Fabric installer, place it in the attached `/data` directory and
```

Reply:
Yes it should; looks like I missed a couple places with my find-replace.